lib/resources/r4/observation.yaml

```diff
@@ -212,12 +223,42 @@ fields:
         }
       ],
     }
+  example2: |
+    {
+      "valueCodeableConcept": {
+                        "coding": [
+                          {
+                            "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                            "code": "OP",
+                            "display": "ordering provider"
+                          }
+                      ],
+                    "text": "ordering provider"
+                },
+                "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+              },
+              {
+                  "valueCodeableConcept": {
+                      "coding": [
+                        {
+                          "system": "http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
+                          "code": "PPRF",
+                          "display": "primary performer"
+                        }
+                        ],
+                "text": "primary performer"
+              },
+              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+            }
+          ],
+          "reference": "Practitioner/2797943"
+        }
   note: |
     <ul>
```

Comment:
@BS062721 unrelated to this PR for the performer field the example is,
```
{
   "reference": 'Practitioner/12724064'
 }
```
Shouldn't it be in `performer` field inside an array like, 
```
{
    "performer": [
        {
            "reference": 'Practitioner/12724064'
        }
    ]
}
```

Reply:
updated: https://github.com/cerner/fhir.cerner.com/commit/1d7546c21c9e25b6edbc48ae90cb38c95ee734b5#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR182-R187